CRM/Export/Form/Select.php

```diff
@@ -193,7 +195,7 @@ public function preProcess() {
     $taskName = $componentTasks[$this->_task];
     $this->assign('taskName', $taskName);
 
-    if ($this->_componentTable) {
+    if ($this->_componentTable && $formName !== 'CRM_Export_StateMachine_Standalone') {
```

Comment:
We must be able to avoid this.... somehow

Reply:
You're welcome to try. The export classes are so complex that I find it difficult to fix things the "right" way because of the risk of breaking something unrelated.

Reply:
This code stumps me a bit. In `\CRM_Contact_Form_Task::preProcessCommon` it explicitly checks to see if this is a standalone form and sets `$this->_componentTable`, and seems to fill that table. So although it's unnecessary to reread from that table when we already have the values in memory (and thus this change improves efficiency if nothing else), I'm not sure why the query returns no results :/